src/app/components/drawer/DrawerRail.js

```diff
@@ -90,6 +106,9 @@ const DrawerRail = (props) => {
             >
               <QuestionAnswerIcon />
             </Link>
+            <button type="button" className={`${styles.railIconButton} ${drawerOpen ? 'side-navigation__toggle-open' : 'side-navigation__toggle-closed'}`} id="side-navigation__toggle" onClick={() => setDrawerOpenChange('feed')}>
```

Comment:
The `ID` of this element is the same as the `button` above... it's better to differentiate. If this selector is needed for any functionality, I suggest to change it to a class.

Reply:
:white_check_mark: